templates/github/.github/workflows/fips.yml.j2

```
@@ -0,0 +1,64 @@
+{% include 'header.j2' %}
+---
+name: Pulp FIPS Nightly CI
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # runs at 4:00 UTC daily
+    - cron: '00 4 * * *'
```

I imagine that not all plugins will want to test against FIPS nightly. What if we introduce a setting (default to true?) and then do something like this:

```suggestion
  {% if fips_test %}
  schedule:
    # * is a special character in YAML so you have to quote this string
    # runs at 4:00 UTC daily
    - cron: '00 4 * * *'
  {% else %}
  workflow_dispatch:
  {% endif %}
```

---

sweet!